assets/scripts/segments/__tests__/scatter.test.js

```diff
@@ -1,22 +1,25 @@
 /* eslint-env jest */
-import { pickObjectsFromPool } from '../scatter'
+import { getRandomObjects } from '../scatter'
 import PEOPLE from '../people.json'
 
-// The unit test for `pickObjectsFromPool()` is designed to ensure that it
+// The unit test for `getRandomObjects()` is designed to ensure that it
 // returns the same values as previous behaviour, because we're heavily
 // refactoring how this works, and want to avoid regressions. We will need to
 // continue making sure that we return similar output against future
 // refactoring -- for instance, when transitioning from imperial to metric
-// units.
+// units. This is why there are many tests where the expected result is a
+// very precise decimal number -- which can be brittle.
 
 // However! The output of this function is not intended to be guaranteed
 // consistent across all releases, because it can change for any reason -- for
 // example, just adding more people to people.json will result in different
-// output. As a future improvement, we can either make this test more general,
-// or roll it up into an integration test.
+// output. Therefore, the brittleness is expected and we should update this
+// test when a number updates for expected reasons. As a future improvement,
```

Comment:
might be nice to include an example of an expected vs unexpected reason, for the uninitiated

Reply:
hmmmm I did but I'll try to reword to be clearer.

Reply:
as a follow up improvement I am adding a mock `people.json` to further limit the chances of the tests breaking unexpectedly. Only when changing the mock input to the test should someone expect tests to break, I think that's a fair tradeoff.